src/schema/comment/giscus.json

```diff
@@ -0,0 +1,116 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "/comment/giscus.json",
+    "description": "Giscus comment plugin configurations",
+    "type": "object",
+    "properties": {
+      "type": {
+        "type": "string",
+        "const": "giscus"
+      },
+      "repo": {
+        "type": "string",
+        "description": "GitHub repository to which Giscus connects"
+      },
+      "repoId": {
+        "type": "string",
+        "description": "GitHub repository id to which Giscus connects"
+      },
+      "category": {
+        "type": "string",
+        "description": "The discussion category where new discussions will be created",
+        "default": "Announcements",
+        "nullable": true
+      },
+      "categoryId": {
+        "type": "string",
+        "description": "Id code of the category of Github discussions which Giscus will access to"
+      },
+      "mapping": {
+        "type": "string",
+        "description": "The mapping between the embedding page and the embedded discussion",
+        "enum": [
+          "pathname",
+          "url",
+          "title",
+          "og:title",
+          "specific",
+          "number"
+        ],
+        "default": "pathname",
+        "nullable": true
+      },
+      "strict": {
+        "type": "integer",
+        "description": "Avoid mismatches due to GitHub's fuzzy searching method when there are multiple discussions with similar titles",
+        "default": false,
+        "nullable": true
+      },
+      "reactionsEnabled": {
+        "type": "boolean",

```

Comment:
Do you want boolean or integer?

Reply:
Boolean should be better since it does not have a third selection.